prowler/lib/check/checks_loader.py

```diff
@@ -77,4 +79,15 @@ def load_checks_to_execute(
                 check_name = check_info[0]
                 checks_to_execute.add(check_name)
 
+    # Verify if any input check is an alias of another check
+    for input_check in checks_to_execute:
+        for check, metadata in bulk_checks_metadata.items():
```

Comment:
Is this the only way to get it? If we have more than 300 checks for AWS, this could result in 9000 iterations. Give this a thought